hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/TenantAwareUserProperties.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2019 devolo AG and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.hawkbit.autoconfigure.security;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import lombok.Data;
+import lombok.ToString;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration for hawwkBit static users.
+ */
+@Data
+@ToString
+@ConfigurationProperties("hawkbit.security.user")
+public class TenantAwareUserProperties {
+
+    private Map<String, User> users = new HashMap<>();
```

Comment:
what is the key of this map supposed to be ? tenant or ?
in some places of code it is taken as username rather then tenant

Reply:
if that is the case maybe better naming of this field would be more clear - something like 'tenantUsers'

Reply:
aah, i see, ok